.gitpod.yml

```diff
@@ -8,7 +8,7 @@ tasks:
       git tag "$(git tag --sort=committerdate | tail -1).dev"
       eval "$(sudo /opt/conda/bin/python -m conda init --dev bash)"
       sudo su root -c "/opt/conda/bin/conda install -yq conda-build"
-      conda-build tests/test-recipes/activate_deactivate_package
+      conda-build tests/test-recipes/activate_deactivate_package tests/test-recipes/pre_link_messages_package
```

Comment:
There are more cases of running conda-build during the test setup, which is why it would make sense to move this line into a Python script (to be cross-platform compatible) into the tests/test-recipes folder and calling it here.

Something like:

```python
import os
import subprocess

test_recipes = os.path.abspath(os.path.dirname(__file__))

packages = [
    os.path.join(test_recipes, package)
    for package in [
        "activate_deactivate_package",
        "pre_link_messages_package",
    ]
]

subprocess.run("conda-build", " ".join(packages), check=True)
```

And then replace the various calls to conda-build with `python tests/test-recipes/build.py` or the analog on Windows.

Reply:
Should we put something similar but as a pytest fixture in ``conftest.py`` with a scope session?
something like:

```
@pytest.fixture(scope="session")
def conda_build_recipes():
    test_recipes = Path(__file__).resolve().parent / "test-recipes"
    recipes_to_build = ["activate_deactivate_package", "pre_link_messages_package"]
    packages = [str(test_recipes / pkg) for pkg in recipes_to_build]
    subprocess.run("conda-build", " ".join(packages), check=True)
```

what do you think?

Reply:
Oooh, I like that! Let's do that <3

Reply:
I think this would live in `tests/conftest.py` then would need to use the `autouse=True` parameter though?

Reply:
yes, you are right, I forgot to add that, thanks :)

Reply:
done!